File: db4on/reference/db4o-reference-chapters/src/com/db4odoc/f1/diagnostics/NewCarModel.cs
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */

using com.db4o.query;

namespace com.db4odoc.f1.diagnostics
{

	public class NewCarModel:Predicate 
	{
		public bool Match(Car car) 
		{
			return car.Model.EndsWith("2002");
		}
	}
}

File: db4on/reference/vb/db4o-reference-chapters/src/com/db4odoc/f1/diagnostics/NewCarModel.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 


Imports com.db4o.query

Namespace com.db4odoc.f1.diagnostics

    Public Class NewCarModel
        Inherits Predicate
        Public Function Match(ByVal car As evaluations.Car) As Boolean
            Return car.Model.EndsWith("2002")
        End Function
    End Class
End Namespace
